Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -351,66 +352,66 @@ AccessibilityRole AccessibilityNodeObject::determineAccessibilityRoleFromNode(Tr
     if (headingLevel())
         return AccessibilityRole::Heading;
 
-    if (element->hasTagName(codeTag))
+    if (elementName == ElementName::HTML_code)
         return AccessibilityRole::Code;
-    if (element->hasTagName(delTag) || element->hasTagName(sTag))
+    if (elementName == ElementName::HTML_del || element->elementName() == ElementName::HTML_s)
```

Comment:
`element->elementName() == ElementName::HTML_s`

Don't we already have an `elementName` variable? Why re-get it?